manifests/prometheus-operator.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: metallb-controller
+spec:
+  selector:
+    matchLabels:
+      component: controller
+  namespaceSelector:
+    matchNames:
+      - metallb-system
+  podMetricsEndpoints:
+    - port: monitoring
+      path: /metrics
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: metallb-speaker
+spec:
+  selector:
+    matchLabels:
+      component: speaker
+  namespaceSelector:
+    matchNames:
+      - metallb-system
+  podMetricsEndpoints:
+    - port: monitoring
+      path: /metrics
```

Comment:
Using the following selector allows reducing this to a single PodMonitor object:
```
  selector:
    matchLabels:
      app: metallb
```

Full example: https://github.com/thaum-xyz/ankhmorpork/blob/master/base/metallb-system/config/podmonitor.yaml